model_compression_toolkit/common/quantization/quantization_params_generation/qparams_activations_computation.py

```diff
@@ -50,7 +51,8 @@ def get_activations_qparams(n: BaseNode,
     else:
         signed = np.any(bins_values < 0)
 
-    if n.prior_info.is_output_bounded():
+    if n.prior_info.is_output_bounded() and \
+            n.activation_quantization_cfg.activation_quantization_method == QuantizationMethod.POWER_OF_TWO:
         n.activation_quantization_cfg.activation_quantization_params_fn = quantization_params_generation.no_clipping_selection_min_max
```

Comment:
I agree with @reuvenperetz  in charge of bounded output. We can use this information to set the threshold or min-max values.